src/Catty.xcodeproj/xcshareddata/xcschemes/Catty (DEBUG).xcscheme

```diff
@@ -57,19 +57,19 @@
             skipped = "NO">
             <BuildableReference
```

Comment:
Please revert your changes to this file.

Reply:
Done!